R/demo.Rmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "demo_plot_forecast"
```

Comment:
I think the right place for a file like this might be in a vignettes folder, not in an R folder